webcompat/api/uploads.py

```diff
@@ -86,6 +86,8 @@ def save(self):
         # Optimize further the image compression for these formats
         if self.image_object.format in ['JPEG', 'JPG', 'JPE', 'PNG']:
```

Comment:
Oh, did we want to remove 'PNG' from the `save_parameters['optimize']` condition?